tests/test_indexer.py

```diff
@@ -160,6 +171,21 @@ def test_es_client_reuse(self):
 
         self.assertIs(es_client_after_first_call, es_client_after_second_call)
 
+    def test_subscription_notification(self):
+        bundle_key = self.load_test_data_bundle_for_path('fixtures/smartseq2/paired_ends')
+        sample_s3_event = self.create_sample_s3_bundle_created_event(bundle_key)
+        process_new_indexable_object(sample_s3_event, logger)
+
+        ElasticsearchTestClient.get().indices.create(DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME)
+        subscribe_for_notification(smartseq2_paired_ends_query,
+                                   "http://green.box.com/notification",
```

Comment:
Let's make this into an example.com URL. `green.box.com` might actually exist. Also, I suggest only allowing https:// URLs.